src/js/view/controls/shortcuts-tooltip.js

```diff
@@ -81,7 +82,7 @@ export default function (container, api, model) {
         api.pause(settingsInteraction);
     };
     const close = () => {
-        shortcutToggle.removeEventListener('click', toggleClickHandler);
+        shortcutToggleUi.off('click tap enter', toggleClickHandler);
```

Comment:
You could just call `shortcutToggleUi.off()` here since there are no other listeners on this UI instance.

Reply:
I hope we call `close` on this when destroying the player and removing controls otherwise we'll have rogue document listeners executing code after the player is removed.

Yep:
1. Setup a player
1b. Add a break point in `documentClickHandler` on ln`if (!/jw-shortcuts|jw-switch/.test(e.target.className)) {`
2. Open the keyboard shortcuts menu
3. Run `jwplayer().remove()` in the console
4. Click the document

Result - this code executes:
```
  var documentClickHandler = function documentClickHandler(e) {
    if (!/jw-shortcuts|jw-switch/.test(e.target.className)) {
      close();
    }
  };
```